examples/typescript/webpack.common.js

```diff
@@ -0,0 +1,71 @@
+const path = require('path');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+const webpack = require('webpack');
+
+module.exports = {
+    entry: {
+        index: './src/index.ts',
+    },
+    plugins: [
+        new webpack.DefinePlugin({
+            WEBSOCKET_URL: JSON.stringify((process.env.WEBSOCKET_URL !== undefined) ? process.env.WEBSOCKET_URL : '')
```

Comment:
Is this websocket URL used somewhere in source code?

Reply:
This has been requested as a want by users however we do not require it in our example specifically. What do you recommend?

Reply:
Yeah, I saw a user request it. But how would a user actually use it? I don't see it used in the examples in this PR for instance.

Reply:
The plan was to have that line included in the webpack but in the updated sps frontend docs include this:

`declare var WEBSOCKET_URL: string;
let signallingServerAddress = WEBSOCKET_URL;
if (signallingServerAddress == '') {
    // define our signallingServerProtocol to be used based on whether
    // or not we're accessing our frontend via a tls
    let signallingServerProtocol = 'ws:';
    if (location.protocol === 'https:') {
        signallingServerProtocol = 'wss:';
    }

    // build the websocket endpoint based on the protocol used to load the frontend
    signallingServerAddress = signallingServerProtocol + '//' + window.location.hostname

    // if the frontend for an application is served from a base-level domain
    // it has a trailing slash, so we need to account for this when appending the 'ws' for the websocket ingress
    if (window.location.pathname == "/") {
        signallingServerAddress += '/ws'
    } else {
        signallingServerAddress += (window.location.pathname + '/ws')
    }
}`
 
This was how users would use it in the old sps frontend interpretation obviously with the new frontend the stuff inside the if would look diffrent.

Reply:
I have resolved this in ded28fd5cd1dce4dc87323e4e9b8626a5bfa2f88 by adding a template demonstration file and this will be covered in the documentation.